projects/gfbio-app/src/app/app.component.ts

```diff
@@ -189,6 +190,11 @@ export class AppComponent implements OnInit, AfterViewInit {
                 icon: 'help',
                 tooltip: 'Help',
             },
+            {
+                sidenavConfig: {component: OidcComponent},
+                icon: 'contacts',
+                tooltip: 'Login',
+            },
```

Comment:
We should not put this below help. Maybe this can be on top.

Reply:
dfd1dac5eab19917b548030c8211d8c565a5cc96